homeassistant/components/lyric/climate.py

```diff
@@ -113,7 +118,9 @@ async def async_setup_entry(
                     ),
                     location,
                     device,
-                    hass.config.units.temperature_unit,
+                    UnitOfTemperature.FAHRENHEIT
+                    if device.units == "Fahrenheit"
+                    else UnitOfTemperature.CELSIUS
```

Comment:
This could/should be moved into the entity itself now (no need to figure that out in this place of the code).

Reply:
Done!